### PR TITLE
Clock moved to new file

### PR DESCRIPTION
**Pull request by salilkapur**

### CODE REVIEW

File: BinPy/tools/clock.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    def kill(self):
+        """
+        Kills the clock(Thread)
+        """
+        self.exitFlag = 1
+
+    def run(self):
+        self.__main_func()
+
```
Comment (debugger22) [inline BinPy/tools/clock.py]: You missed a newline here.

Comment (salilkapur) [inline BinPy/tools/clock.py]: Correcting!

**Comment by debugger22** [comment]
It's interesting that how this test got affected by this change.

```
FAIL: BinPy.tests.series_4000_tests.test_IC_4022
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/BinPy/BinPy/BinPy/tests/series_4000_tests.py", line 421, in test_IC_4022
    0, 10: 0, 11: 0, 12: 0}
AssertionError
```

**Comment by salilkapur** [comment]
Strangely it does not fail on my system.

**Comment by debugger22** [comment]
And it also passed with the next commit. :smile:

**Pull request merged**